web/frontend/src/components/slam/2d-renderer.vue

```diff
@@ -147,7 +147,7 @@ const updatePose = (newPose: commonApi.Pose) => {
   baseMarker.position.set(x, y, z);
 
   const theta = THREE.MathUtils.degToRad(newPose.getTheta());
-  baseMarker.geometry.rotateZ(theta);
+  baseMarker.material.rotation = theta;
```

Comment:
[q] how do we know which axis this rotates? Can you add a comment abt this?
I know that recently slam underwent the transition from displaying in XZ to XY, in the case of another transition I think a comment would be useful context :)

Reply:
Sprites can only rotate on an axis directly orthogonal to the camera. So in other words sprites cannot rotate freely in three-dimensional space; their rotation is limited to a specific plane that is parallel to the screen.

I'd normally agree with adding a comment but since this is general education sort of stuff and not specific to our code I'm not sure if I'd include it. Thoughts?

Reply:
ok this makes sense to omit as a comment.